data/sql/updates/pending_db_world/rev_1662444495729947700.sql

```diff
@@ -0,0 +1,7 @@
+DELETE FROM `phase_area` WHERE `AreaId`=4806 AND `PhaseId`=181;
+INSERT INTO `phase_area` (`AreaId`, `PhaseId`, `Comment`) VALUES 
+(4806, 181, 'Phase for quest 14400');
+
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId`=26 AND `SourceGroup`=181 AND `SourceEntry`=4806 AND `SourceId`=0 AND `ElseGroup`=0;
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES 
+(26, 181, 4806, 0, 0, 9, 0, 14400, 0, 0, 0, 0, 0, '', 'Phase only if quest 14400 is taken');
```

Comment:
It Looks like great!, what do you think @pangolp ?

Reply:
> > It Looks like great!, what do you think @pangolp ?
> 
> The truth is that it looks good, we had passed something similar, but instead of 26, it was 23, but I think it was practically the same, it happens that the way it is written in the .h file, is a little confusing. But if it works, which, I could not test it, for me there is no problem.

23 is CONDITION_SOURCE_TYPE_NPC_VENDOR                     = 23,

Reply:
> > > It Looks like great!, what do you think @pangolp ?
> > 
> > 
> > The truth is that it looks good, we had passed something similar, but instead of 26, it was 23, but I think it was practically the same, it happens that the way it is written in the .h file, is a little confusing. But if it works, which, I could not test it, for me there is no problem.
> 
> 23 is CONDITION_SOURCE_TYPE_NPC_VENDOR = 23,

Yes I know, we saw it with Jose the other day, but as the file is a bit confusing, and we have not yet taken this information to the wiki, they gave us this example with the 23. I am finishing solving the dummy issue and trying other pull request, but in the worst case, if it is wrong, it is reversed and that's it.

Reply:
I was testing with 23 and is not working, but if you test with 26 it works @pangolp